src/website/views/templates/autopay.pug

```diff
@@ -0,0 +1,315 @@
+---
+title: Autopay
+---
+
+- var days = ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23', '24', '25', '26', '27', '28', '29', '30', '31'];
+- var years = ['2019', '2020', '2021', '2022', '2023', '2024', '2025', '2026', '2027', '2028', '2029', '2030'];
+
+p.-text Basic Autopay page template. Indicates several ways to automate the way to pay.
+
+.example.-mb--3.-bg--grey-10
+  .chi-app.-p3
+    .m-epanel.-bordered.-p--1#autopay
+      .m-epanel__header.-d--flex.-justify-content--between
+        div
+          h5.-m--0.a-card__title Automatic Payments
+          p.-text Each month, an automatic payment for the full amount of your bill will be paid to CenturyLink from the payment method of your choosing.
+        .m-formGroup
+          input(type="checkbox", class="a-input -toggle", id="toggle1", data-chi-epanel-action='toggle')
+          label(for="toggle1")
```

Comment:
Can we add an Autopay example that's identical to this version without a toggle switch and call it "Base" and then call this second version "Optional"?

Reply:
done